pkg/services/rediscache/plan_schemas.go

```diff
@@ -20,6 +24,119 @@ func generateProvisioningParamsSchema() service.InputParametersSchema {
 				Description: "The (new or existing) resource group with which" +
 					" to associate new resources.",
 			},
+			"enableNonSslPort": &service.StringPropertySchema{
+				Title:         "Enable non-SSL port",
+				Description:   "Specifies whether the non-ssl Redis server port (6379) is enabled.",
+				AllowedValues: []string{"enabled", "disabled"},
+				DefaultValue:  "enabled",
+			},
+			"skuCapacity": &service.IntPropertySchema{
+				Title:         "Sku capacity",
```

Comment:
nit: I believe that it should be SKU (throughout this PR) because it is an acronym.

Reply:
Thanks, will fix it.